vuepress/docs/next/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -52,30 +51,25 @@ Bundles are generated using the `ent bundle` command and its convenience methods
    customUi: <h2>Hi from Example Widget</h2>
    ```
 
-## Publish the Bundle
-
-1. From the root bundle directory, generate the Docker image:
+5. From the root bundle directory, generate the Docker image:
    ``` sh
    ent bundle pack
    ```
-   The `pack` command builds the bundle components, then constructs a Docker image for the bundle.
+   The `pack` command builds the widget, then constructs a Docker image for the bundle.
 
-2. Publish the Docker image to a Docker registry:
+6. Publish the Docker image to a Docker registry:
    ``` sh
    ent bundle publish
    ```
-   When prompted, input the organization of your choice, e.g. the user ID of your Docker account.
 
-3. Deploy the bundle to your Entando Application using the bundle Docker URL.
+7. Deploy the bundle to your Entando Application:
    ``` sh
-   ent ecr deploy --repo=docker://registry.hub.docker.com/YOUR-DOCKER-ORG/YOUR-BUNDLE-NAME
+   ent bundle deploy
    ```
-## Install the Bundle into an Application
-
-1. In your Entando instance, go to `App Builder` → `Component Repository` 
+   Your bundle will appear in the Local Hub of your Entando instance, accessible from `App Builder` → `Hub`, and show a status of DEPLOYED.
 
-2. Click `Install`
-
-3. Verify the `Install` button changes to `Uninstall` to indicate that the installation completed successfully
-
-4. Go to `App Builder` → `Components` → `Micro Frontends & Widgets` to confirm that your bundle appears in the "User" section
```

Comment:
Might be worth a note that you can either install from App Builder or via the CLI here.